SU-SWS/saa_victoria_subtheme
# ADAPT-000: Removed red underline for regular links not hovered on.

# READY FOR REVIEW

# Summary
- remove red underline from regular links when not hovered/focused on 

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. check a regular link that it doesn't have underline regularly. it should still have underline when hovered on.
4. check other links with icon classes, they shouldn't change (no underline, then underline on hover)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
